extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -196,12 +202,81 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				}
 			}
 		});
-
 		this._assessmentResults = {
 			issues: serverLevelAssessments,
 			databaseAssessments: databaseLevelAssessments
 		};
 
+		try {
+			const queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(connectionProfile.providerId, azdata.DataProviderType.QueryProvider);
+			const query = 'SELECT name, compatibility_level FROM sys.databases; ';
```

Comment:
I would strongly suggest putting all of this into an async function and not block this logic on it for perf reasons